src/services/GasService.sol

```diff
@@ -22,6 +22,9 @@ contract GasService is ReentrancyGuardUpgradeable, OwnableUpgradeable, IGasServi
   ConfigStorage public configStorage;
   uint256 public executionFeeInUsd;
   address public executionFeeTreasury;
+  bool public isExecutionFeeSubsidization;
```

Comment:
Don't think we need this variable as we can adjust `waivedExecutionFeeTradeSize` to 0 to disable